docs/ci-cd-integration/deploy-from-github.md

```diff
@@ -131,7 +131,20 @@ You will need to add the following information into GitHub Secrets:
 You can add GitHub Secrets using the instructions [here](#add-the-github-secrets)
 
 #### Add the Captain Definition File
-Add the `captain-definition` file described [here](#add-files-to-project)
+Add the `captain-definition` file described [here](#add-files-to-project). If are going to be using the github action below, the `captain-definition` contents should be:
+
+```
+ {
+  "schemaVersion": 2,
+  "imageName": "ghcr.io/<YOUR_GITHUB_USERNAME>/<REPOSITORY_NAME>:latest"
+ }
+```
+
+This will ensure your Caprover app is using the latest version of the image you just built in the GitHub Action.
+
+#### Add a private Docker Registry to CapRover
```

Comment:
:+1:  Thanks for adding this section!

Let's add more details:

```
Use these values:
username: <your github username>
Password: <your github personal token>
Domain: ghcr.io (no www, no http)
Image Prefix: <your github username or your org username (if you're pulling images from an org different than your username)>
```

Reply:
Good idea, done! 🚀